pkg/lib/journal.js

```diff
@@ -230,7 +234,34 @@
                 };
             if (count > 1)
                 parts['count'] = count;
-            if (prio < 4)
+            if (ident === 'abrt-notification') {
+                // Problem can be identified by 3 elements. Most likely the first, but in some
+                // cases other two can be used as well.
+                var all_p = [entry['PROBLEM_DIR'], entry['PROBLEM_DUPHASH'], entry['PROBLEM_UUID']];
+                var p;
+                var known = false;
+                for (var i = 0; i < all_p.length; i++) {
+                    if (all_p[i] in displayable_problems) {
+                        p = all_p[i];
+                        known = true;
+                        break;
+                    }
+                }
+                //Do not render unknown (deleted) problems
+                if (! known)
+                    return "";
```

Comment:
In the design, I intended the counter only for things happening right after each other (to reduce noise from services that fails constantly and spits out lines), so it's consistent with how the rest of the logs page work. Sorry if the design was unclear there.

For situations where a crash happens, say, once a day, I feel it's important that you see that it's reoccurring on a predictable time-basis, and that the entry's for different days don't get collapsed down.

I agree with the idea of the notification area. It will be needed also for things like software updates being available and other critical things, so let's solve that as part of that.